CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 
 ## [Unreleased]
 
+## [0.4.28] - 2025-09-02
+
+## What's Changed
+* ci: drop really old trick and ensure MSRV for all feature combo by @tisonkun in https://github.com/rust-lang/log/pull/676
+* chore: fix some typos in comment by @xixishidibei in https://github.com/rust-lang/log/pull/677
+* Unhide `#[derive(Debug)]` in example by @ZylosLumen in https://github.com/rust-lang/log/pull/688
```

Comment:
I assume it's automatically generated, but I don't think fixing typos and changing something in an example should be called out in the change log. For someone who is reading this with the indent to understand what changed before updating I think this is just noise.

Reply:
This was automatically generated. I'm happy to massage the changelog file a bit to focus on changes users may want to be aware of